BEST_PRACTICES.md

```diff
@@ -121,85 +138,60 @@ The following are options that can be passed into the httpMiddlewareOption when
 
 With all the above options, an optimal combination of retries and request timeouts can be properly configured. Also it is important to note that the values of these parameters are chosen in such a way that it doesn’t affect the experience of the SDK. e.g if requests times out frequently due to connection or networking issues then a higher value for timeout and a lower value for retryDelay can be chosen.
 
-## How exactly the customers could best utilize the logging feature in our SDKs.
-
-The JS/TS SDK has a logger middleware called the withLoggerMiddleware() which can be chained at multiple parts when building the client. The logger middleware logs data like the request headers, body, request and response.
-
-```ts
-const client = new ClientBuilder()
-  .withProjectKey(projectKey)
-  .withClientCredentialsFlow(authMiddlewareOptions)
-  .withLoggerMiddleware() // Log the request/response at this point
-  .withHttpMiddleware(httpMiddlewareOptions)
-  .withUserAgentMiddleware() // Include data from the http-middleware
-  .withLoggerMiddleware()
-  .build()
-```
-
```

Comment:
Any reason why this code block was this removed?

Reply:
It's redundant there is already the section regarding the logging